### PR TITLE
 Task #129281 fix: Backend UI fix, package warning fix, variable name conflict

### PR DESCRIPTION
**Pull request by monikaatvowel**

### CODE REVIEW

File: src/components/com_tjdashboard/administrator/sql/install.mysql.utf8.sql [hunk 1]
```diff
@@ -50,12 +40,3 @@ CREATE TABLE IF NOT EXISTS `#__tj_dashboard_widgets` (
   FOREIGN KEY (`dashboard_id`) REFERENCES `#__tj_dashboards` (`dashboard_id`)
 );
 
---
--- Indexes for table `#__tj_dashboard_widgets`
```
Comment (thite-amol) [inline src/components/com_tjdashboard/administrator/sql/install.mysql.utf8.sql]: @twsvaishali Can you check why this is required?

cc
@coolbung

File: src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php [hunk 2]
```diff
@@ -14,7 +14,8 @@
 
 $document->addScript(Juri::root() . 'plugins/tjdashboardrenderer/tabulator/assets/js/jquery-ui.min.js');
 $document->addStyleSheet(JUri::root() . '/media/techjoomla_strapper/bs3/css/bootstrap.css');
-$document->addStyleSheet(JUri::root() . 'media/com_tjdashboard/css/font-awesome.min.css');
+$document->addStyleSheet(JUri::root() . 'media/com_tjlms/font-awesome/css/font-awesome.min.css');
+$document->addStyleSheet(JUri::root() . 'media/com_tjlms/css/tjlms_backend.css');
```
Comment (thite-amol) [inline src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php]: @monikaatvowel Why we have tjlms dependency here?

Comment (twsvaishali) [reply src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php]: @monikaatvowel  you will need to add the condition in the onAfterRoute function of tjlms system plugin. We alredy have code for jlike and categories written there

Comment (monikaatvowel) [reply src/components/com_tjdashboard/administrator/views/dashboard/tmpl/default.php]: done. removed from here and added there

**Pull request merged**